src/effects/backends/builtin/metronomeeffect.cpp

```diff
@@ -124,7 +123,7 @@ void MetronomeEffect::loadEngineEffectParameters(
 }
 
 void MetronomeEffect::processChannel(
-        MetronomeGroupState* pGroupState,
+        MetronomeGroupState* pGs,
```

Comment:
pls no

I'd be okay with `state` if you want to shorten it, since there's only one state in this file.

Reply:
we already call it `gs` below. This just aligns with our code convention of prefixing pointers with `p`. I while not particularly necessary I don't have anything against this check.

Reply:
I reverted back to pGroupState. The short hand was anyway confusing.